### ISSUE TITLE
parcel build fails to auto-install and locate installed packages

### ISSUE THREAD

**Issue by ubersl0th** [parcel-bundler/parcel#7095]
I'm trying to build a React/TypeScript/SCSS library with parcel following the library guide.
When running `parcel build` I get the following error message:
```
Failed to install @parcel/transformer-typescript-types: Callback must be a function. Received undefined
```

So I tried to install the packages manually, and afterwards my package.json devDeps contains the following:
```json
"@parcel/packager-ts": "^2.0.0",
"@parcel/transformer-sass": "^2.0.0",
"@parcel/transformer-typescript-types": "^2.0.0",
...
"parcel": "^2.0.0",
...
```

Then if I run `parcel build --no-autoinstall` I get the following error:
```
@parcel/core: Cannot find Parcel plugin "@parcel/transformer-typescript-types"
```

Not sure what to do here, any help at all would be greatly appreciated

**Comment by yGuy** [comment]
I was about to post a very similar problem. 
I tried using "data-url" inlining, but auto-install failed me with the following message:
```
Error: Failed to install @parcel/transformer-inline-string: Yarn failed to install modules:Child process failed
````
(and I was not able to switch to npm, either - it always used yarn (in the error message, at least), even when I removed my node modules and all package lock files and reinstalled with npm.)

And with `--no-autoinstall` and after a manual installation of `@parcel/transformer-inline-string` and `@parcel/optimizer-data-url`
I am now getting this:

```
@parcel/core: Cannot find Parcel plugin "@parcel/transformer-inline-string"
```
And I have no idea what to do next. 
There is no yarn-error.log and the message is pretty generic.
